ui/imports/shared/panels/DidYouKnowSplashScreen.qml

```diff
@@ -28,6 +28,13 @@ Pane {
                 anchors.bottom: parent.bottom
                 width: parent.width
                 visible: root.progress != 0
```

Comment:
I know, it was there but, `!==` :)

Reply:
> And, as a fun, https://www.qt.io/blog/2018/05/29/playing-coroutines-qml could be used here :D

This is really nice. Haven't played with it so far. I'll keep it this way for now.
Thanks @micieslak. I've updated the PR.